app/services/export_file_attachments.rb

```diff
@@ -0,0 +1,68 @@
+class ExportFileAttachments
+  attr_reader :source_file, :blob
+
+  TEMP_FILE_PATH = "foo.zip"
+  BULLETIN_LIST_URL = "https://www.apprenticeship.gov/about-us/legislation-regulations-guidance/bulletins/export?search=&category%5B0%5D=National%20Guideline%20Standards&category%5B1%5D=National%20Program%20Standards&category%5B2%5D=Occupations&page&_format=csv"
+
+  def initialize(source_file, blob)
+    @source_file = source_file
+    @blob = blob
+  end
+
+  def call
+    zip_file = create_zip_version_of_source_file
+    file_names = unzip_attachments_and_list_file_names
+    save_attachments_to_db(file_names)
+    delete_extracted_files(file_names, zip_file)
+  end
```

Comment:
How are we handling exceptions in the project? I wanted to add a generic rescue here, but don't know what would do in the treatment stage.